### ISSUE TITLE
False positives on doctrine-orm-dangerous-query

### ISSUE THREAD

**Issue by Sjord** [returntocorp/sgrep-rules#1491]
Some things that aren't related to Doctrine at all match the php.doctrine.security.audit.doctrine-orm-dangerous-query.doctrine-orm-dangerous-query.

For example, when running on [this project](https://github.com/kaltura/server), some false positives:

```
vendor/symfony/view/sfView.class.php
314:    $this->parameterHolder->add(sfConfig::get('mod_'.strtolower($moduleName).'_view_param', array()));

vendor/tfpdf/font/unifont/ttfonts.php
624:		$this->add('post', $post);

vendor/symfony/vendor/phing/tasks/system/CvsTask.php
466:            $c->createArgument(true)->setValue("-z" . $this->compression);

vendor/symfony/i18n/Gettext/PO.php
103:            $this->strings[parent::prepare($msgid)] = parent::prepare($msgstr);
--------------------------------------------------------------------------------
103:            $this->strings[parent::prepare($msgid)] = parent::prepare($msgstr);
--------------------------------------------------------------------------------
149:                'msgid "'  . parent::prepare($o, true) . '"' . "\n" .
--------------------------------------------------------------------------------
150:                'msgstr "' . parent::prepare($t, true) . '"' . "\n\n"

vendor/symfony/addon/propel/sfPropelBehavior.class.php
64:          sfConfig::set('propel_behavior_'.$name.'_'.$class.'_'.$key, $value);

plugins/content_distribution/providers/ContentDistributionServiceBase.php
157:					$cacheStore->set($cacheKey . self::CACHE_CREATION_TIME_SUFFIX, time(), self::CACHE_EXPIRY);
--------------------------------------------------------------------------------
158:					$cacheStore->set($cacheKey, $xml, self::CACHE_EXPIRY);

alpha/apps/kaltura/lib/clipconcat/kClipManager.php
286:		if (!$store || $store->add($lockKey, true, self::LOCK_EXPIRY))
```

**Comment by Sjord** [comment]
Ping @inkz.

**Comment by inkz** [comment]
yep, noticed it few hours ago :smile: 
thank you, going to fix soon :v:
